addons/mrp/models/mrp_production.py

```diff
@@ -934,7 +931,7 @@ def _get_move_finished_values(self, product_id, product_uom_qty, product_uom, op
             'operation_id': operation_id,
             'byproduct_id': byproduct_id,
             'name': _('New'),
-            'date': self._get_date_planned_finished(),
+            'date': self._get_date_finished(),
```

Comment:
It's not possible to use self.date_finished directly? And remove the function

Reply:
this was not possible because the date_finished was missing when creating the first finished move.
a default has been introduced and the function removed